.travis.yml

```diff
@@ -1,15 +1,13 @@
 language: python
+sudo: false
```

Comment:
I don't think this is required

Reply:
Yeah, check out the [Config tab](https://travis-ci.org/github/zopefoundation/zc.catalog/builds/686485939/config) on the Travis build for this PR:

> <code>root</code>: deprecated key <code>sudo</code> (The key `sudo` has no effect anymore.)

Reply:
It is deprecated. Check the [Travis build config](https://travis-ci.org/github/zopefoundation/zc.catalog/builds/686485939/config) and click "View config" to see the deprecation warning.

Reply:
I changed it.